docs/notifications.md

```diff
@@ -38,13 +38,39 @@ notify:
      content: '{"color":"green","message":"New package published: * {{ name }}*","notify":true,"message_format":"text"}'
 ```
 
+### Publisher information
+
+You can access to the package publisher information in the `content` of a webhook using the `publisher` object.
+
+See below the `publisher` object type:
+
+```
+{
+  name: string,
+  groups: string[],
+  real_groups: string[]
+}
+```
+
+An example:
+
+```
+notify:
+  method: POST
+  headers: [{'Content-Type': 'application/json'}]
+  endpoint: https://usagge.hipchat.com/v2/room/3729485/notification?auth_token=mySecretToken
+  content: '{"color":"green","message":"New package published: * {{ name }}*. Publisher name: * {{ publisher.name }} *.","notify":true,"message_format":"text"}'
+```
+
+**Note:** it's not possible to get the publisher information if the `package.json` file already has the `publisher` property.
+
 ## Configuration
 
-Property | Type | Required | Support | Default | Description
---- | --- | --- | --- | --- | ---
-method| string | No | all |  | HTTP verb
-packagePattern| string | No | all |  | Only run this notification if the package name matches the regular expression
-packagePatternFlags| string | No | all |   | Any flags to be used with the regular expression
-headers| array/object | Yes | all |  | If this endpoint requires specific headers, set them here as an array of key: value objects.
-endpoint| string | Yes | all |  | set the URL endpoint for this call
-content| string | Yes | all |  | any [handlebar](https://handlebarsjs.com/) expressions
+| Property            | Type         | Required | Support | Default | Description                                                                                  |
+| ------------------- | ------------ | -------- | ------- | ------- | -------------------------------------------------------------------------------------------- |
```

Comment:
All those white spaces need to be clean up

Reply:
fixed